client/src/pages/Stories/Stories.js

```diff
@@ -4,7 +4,136 @@ import './Stories.css';
 class Stories extends Component {
     render() {
         return (
-            <h1>Our Stories</h1>
+            <div>
+                <h1>Our Stories</h1>
+                <img></img>
+                <ul class="flex-container">
+                    <li>
+                        <img></img>
+                        <p>
+                            Lorem ipsum dolor sit amet, consectetur adipiscing 
+                            elit, sed do eiusmod tempor incididunt ut labore et
+                            dolore magna aliqua. Ut enim ad minim veniam, quis
+                            nostrud exercitation ullamco laboris nisi ut 
+                            aliquip ex ea commodo consequat. Duis aute irure 
+                            dolor in reprehenderit in voluptate velit esse 
+                            cillum dolore eu fugiat nulla pariatur. Excepteur
+                            sint occaecat cupidatat non proident, sunt in 
+                            culpa qui officia deserunt mollit anim id est 
+                            laborum.
+                        </p>
```

Comment:
Could you try creating a separate component and then importing that for each li item?

Reply:
The changes below are fine, but what did you have in mind for these? Presumably I would need .js and .css files, but I have no understanding of how React is used to return the elements there.